dbt/adapters/bigquery/connections.py

```diff
@@ -712,11 +712,6 @@ def _sanitize_label(value: str) -> str:
     value = _SANITIZE_LABEL_PATTERN.sub("_", value)
     value_length = len(value)
```

Comment:
I don't think you need to check the length of value before slicing. You should be able to just do this:
```
value = _SANITIZE_LABEL_PATTERN.sub("_", value)
return value[:_VALIDATE_LABEL_LENGTH_LIMIT]
```

Reply:
Great suggestion, thank you, resolved in https://github.com/dbt-labs/dbt-bigquery/pull/442/commits/cc57d35ddeac553e1a0d67176601f698783cbe8b